multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/ProcessLogger.java

```diff
@@ -77,7 +79,13 @@ public OperationLogEntry getOperationLogEntry() {
     }
 
     private void createLogMessage(Object message, Level logLevel) {
-        logMessage = layout.toSerializable(createEvent(message, logLevel));
+        String formattedLogMessage = layout.toSerializable(createEvent((message), logLevel));
+        byte[] header = layout.getHeader();
+        if (header != null && !headerIsSet) {
```

Comment:
Why this is not part of the other method?

Can we rely on having `header != null` then use it, without additional parameter `headerIsSet`?

There is one more createLogMessage method that should use header as well.